UnicodeJsps/src/main/java/org/unicode/jsp/UnicodeJsp.java

```diff
@@ -420,7 +421,7 @@ public static String getVersions() {
                     : "<p style='border: 1pt solid red;'>Properties use ICU for Unicode V"
                             + UCharacter.getUnicodeVersion().getVersionString(2, 2)
                             + "; the beta properties support Unicode V"
-                            + CachedProps.CACHED_PROPS.version.getVersionString(2, 2)
+                            + VersionInfo.getInstance(Settings.latestVersion).getVersionString(2, 2)
```

Comment:
String --> VersionInfo --> String

could just use Settings.latestVersion directly if we are ok with printing an extra ".0"

Reply:
I had that originally but thought the extra ".0" was inconsistent so switched it to `VersionInfo...getVersionString()` to get it consistent. I am going to leave it for this round of work.